packages/mdc-rtl/README.md

```diff
@@ -28,76 +26,22 @@ npm install --save @material/rtl
 
 ## Usage
 
-Simply `@import "@material/rtl/mixins";` and start using the mixins. Each mixin is described below.
+### Sass Mixins
 
-### mdc-rtl
+`mdc-rtl` is the most flexible mixin, because it can work with multiple CSS properties. All other RTL mixins logic could be engineered by only using `mdc-rtl`, but we provide these mixins for convenience.
 
-```scss
-@mixin mdc-rtl($root-selector: null)
-```
-
-Creates a rule that will be applied when an MDC-Web component is within the context of an RTL layout.
-
-Usage Example:
-
-```scss
-.mdc-foo {
-  position: absolute;
-  left: 0;
+Both `mdc-rtl-reflexive-property` and `mdc-rtl-reflexive-box` work with one base box-model property, e.g. margin, border, padding. But `mdc-rtl-reflexive-property` is more flexible because it accepts different left and right values. `mdc-rtl-reflexive-box` assumes the left and right values are the same, and therefore that the box-model is symmetrical.
 
-  @include mdc-rtl {
-    left: auto;
-    right: 0;
-  }
+`mdc-rtl-reflexive-position` is the least flexible mixin. It only works with one horizontal position property, "left" or "right". It also assumes the left and right values are the same.
 
-  &__bar {
-    margin-left: 4px;
-    @include mdc-rtl(".mdc-foo") {
-      margin-left: auto;
-      margin-right: 4px;
-    }
-  }
-}
-
-.mdc-foo--mod {
-  padding-left: 4px;
-
-  @include mdc-rtl {
-    padding-left: auto;
-    padding-right: 4px;
-  }
-}
-```
+| Mixin | Description |
+| ----------------------------------------------- | - |
+| `mdc-rtl($root-selector)` | Creates a rule that is applied when the root element is within an RTL context |
```

Comment:
nit: Create rules applied within `$root-selector` scope when the root element is contained within an RTL context